string link_name
---
float32[] pose
